glean-core/python/glean/_process_dispatcher.py

```diff
@@ -123,7 +123,7 @@ def dispatch(cls, func, args) -> Union[_SyncWorkWrapper, subprocess.Popen]:
             # Explicitly pass the contents of `sys.path` as `PYTHONPATH` to the
             # subprocess so that there aren't any module search path
             # differences.
-            python_path = ":".join(sys.path)
+            python_path = (";" if os.name == "nt" else ":").join(sys.path)
```

Comment:
Python exposes this as a constant!

```suggestion
            python_path = os.pathsep.join(sys.path)
```